components/symbl_ai/actions/post-video-url/post-video-url.mjs

```diff
@@ -4,20 +4,86 @@ export default {
   key: "symbl_ai-post-video-url",
   name: "Submit Video URL",
   description: "Submit a Video by providing the URL for processing. See the doc [here](https://docs.symbl.ai/docs/async-api/overview/video/post-video-url)",
-  version: "0.0.2",
+  version: "0.0.3",
   type: "action",
   props: {
     symblAIApp,
     videoUrl: {
       type: "string",
       label: "Video URL",
-      description: "The URL of the video to be processed",
+      description: "The URL of the video to be processed.",
       optional: false,
     },
     meetingName: {
       type: "string",
       label: "Meeting Name",
-      description: "The meeting name. The default name is set to the conversationId",
+      description: "The meeting name. The default name is set to the conversationId.",
+      optional: true,
+    },
+    customVocabulary: {
+      type: "string[]",
+      label: "Custom Vocabulary",
+      description: "List of words and phrases that provide hints to the speech recognition task.",
+      optional: true,
+    },
+    confidenceThreshold: {
+      type: "string",
+      label: "Confidence Threshold",
+      description: "Minimum confidence score that you can set for an API to consider it as a valid insight (action items, follow-ups, topics, and questions). It should be in the range <=0.5 to <=1.0 (i.e., greater than or equal to 0.5 and less than or equal to 1.0.). The default value is 0.5.",
+      optional: true,
+    },
+    detectPhrases: {
+      type: "string",
+      label: "Detect Phrases",
+      description: "It shows Actionable Phrases in each sentence of conversation. These sentences can be found using the Conversation's Messages API. Accepts `true` or `false` values.",
+      optional: true,
+    },
+    webhookUrl: {
+      type: "string",
+      label: "Webhook URL",
+      description: "Webhook URL on which job updates to be sent.",
+      optional: true,
+    },
+    detectEntities: {
+      type: "string",
+      label: "Detect Entities",
+      description: "It returns any entities detected in the conversation. See [Entities API](https://docs.symbl.ai/docs/conversation-api/entities) for reference. Default value is false.",
+      optional: true,
+    },
+    languageCode: {
+      type: "string",
+      label: "Language Code",
+      description: "Language used in the conversation. See [supported languages](https://docs.symbl.ai/docs/async-api/overview/async-api-supported-languages) for reference. Default language is English (en-US).",
+      optional: true,
+    },
+    mode: {
+      type: "string",
+      label: "Mode",
+      description: "Set this parameter to `phone` when the audio is generated from a phone call (8khz sampling rate). The `default` mode works for audio generated from a video or audio meeting (16khz or higher sampling rate).",
+      optional: true,
+    },
+    enableSeparateRecognitionPerChannel: {
+      type: "string",
```

Comment:
Since `enableSeparateRecognitionPerChannel` only accepts `true` or `false`, it should be boolean instead of string.
```suggestion
      type: "boolean",
```

Reply:
Addressed in 1ec7d858